### DISCUSSION TITLE
Has anyone utilized 3-d models as markers or custom layers?

### DISCUSSION THREAD

**Discussion by tracilevine** [uber/react-map-gl#1850] [Q&A]
Any advice would be helpful? Looking to creating a custom map layer with a 3d model as the markers
?

I am looking for ways to implement:  https://visgl.github.io/react-map-gl/docs/api-reference/layer 

With: https://docs.mapbox.com/mapbox-gl-js/example/add-3d-model/. 

I am currently able to use  the 3-d model as a custom marker but it will only render one model at a time.

**Comment by ibgreen** [comment]
An option is https://deck.gl/docs/api-reference/mesh-layers/scenegraph-layer
